managed/services/telemetry/config.go

```diff
@@ -127,17 +135,23 @@ var defaultConfig string
 func (c *ServiceConfig) Init(l *logrus.Entry) error { //nolint:gocognit
 	c.l = l
 
-	telemetry, err := c.loadConfig()
+	var configFile string
+	configFileFromEnv, present := os.LookupEnv(envConfigFile)
+	if present {
+		configFile = configFileFromEnv
+	}
```

Comment:
Minor: It can be just 
```suggestion
	configFile := os.GetEnv(envConfigFile)
```

Reply:
Yeah, in this case it is equivalent, thx